specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -1195,16 +1191,16 @@ via annotations discovered in requirement 2.
 . The deployment tool must allow the Deployer
 to deploy the same module multiple times, as multiple independent
 applications, possibly with different configurations. For example, the
-enterprise beans in an ejb-jar file might be deployed multiple times
+enterprise beans in an enterprisebean-jar file might be deployed multiple times
```

Comment:
again, this artifact should probably be left as ejb-jar.

Reply:
done